README.md

```diff
@@ -429,27 +429,33 @@ HTTPS parameters
 
 - `https`: *[Object]* Run a HTTPS server using Roosevelt.
   - Object members:
-  - `enable`: Enable HTTPS server.
+  - `enable`: Enable a HTTPS server.
     - Default: *[Boolean]* `false`.
-  - `httpsOnly`: Disable HTTP server when running a HTTPS server.
+  - `httpsOnly`: Disable the HTTP server when running a HTTPS server.
     - Default: *[Boolean]* `false`.
-  - `httpsPort`: The port your app will run a HTTPS server on.
+  - `httpsPort`: The port your app will run a HTTPS server on. Can also be defined using the `HTTPS_PORT` environment variable.
```

Comment:
Let's change this from `httpsPort` to `port`, for the sake of sanity

Reply:
Done!